src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -859,21 +877,54 @@ To get all submitted job IDs and names, use `list_submitted_jobs` pragma this wa
 #%list_submitted_jobs()
 ```
 
-==== Print results
+==== Export the workflow in xml format
 
-To get the job result(s), the user has to use the `#%get_result()` pragma by providing the job name:
+To export the created workflow in _.xml_ format, use the `#%export_xml()` pragma:
 
 ```python
-#%get_result(name=JOB_NAME)
+#%export_xml([name=FILENAME])
 ```
 
-Or, by the job ID:
+Notice that the _.xml_ file will be saved under one of the following names:
+
+1. The parameter `name` 's value, if provided
+2. The job's name, if created
+3. The notebook's name, if the kernel can retrieve it
+4. `Unnamed_job`, otherwise.
+
+==== Get results
+
+After the execution of a ProActive workflow, two outputs information can be obtained,
+* results: values that have been saved in the
+https://doc.activeeon.com/latest/user/ProActiveUserGuide.html#_task_result[task result variable],
+* console outputs: classic outputs that has been displayed/printed
+
+To get task results, use the `#%get_task_result()` pragma by providing the task name, and one of the job ID or name:
+
+```python
+#%get_task_result([job_id=JOB_ID], [job_name=JOB_NAME], task_name=TASK_NAME)
+```
+
+To get the result(s) of all the tasks of a job, the user has to use the `#%get_job_result()` pragma by providing the
+job name or the job ID:
 
 ```python
-#%get_result(id=JOB_ID)
+#%get_job_result([job_id=JOB_ID], [job_name=JOB_NAME])
 ```
 
-The returned values of your final tasks will be automatically printed.
+Same behavior of the Kernel to display console outputs. To print a task output, you can use the
+`#%print_task_output()` pragma:
+
+```python
+#%print_task_output([job_id=JOB_ID], [job_name=JOB_NAME], task_name=TASK_NAME)
+```
+
+and finally, to print all job outputs, the user may use the `#%print_job_output()` pragma by providing the
```

Comment:
same here "user may use" bugs me a bit.
Suggestion:
"Finally, the `#%print_job_output()` pragma allows to print all job outputs, by providing..."